client/landing/gutenboarding/onboarding-block/style-preview/font-select.tsx

```diff
@@ -51,10 +51,10 @@ const FontSelect: React.FunctionComponent = () => {
 			<Button
 				className={ classnames( 'style-preview__font-option', {
 					'is-selected':
-						selectedFonts?.headings === selectedDesignDefaultFonts?.[ 0 ] &&
-						selectedFonts?.base === selectedDesignDefaultFonts?.[ 1 ],
+						selectedFonts?.headings === selectedDesignDefaultFonts?.headings &&
+						selectedFonts?.base === selectedDesignDefaultFonts?.base,
```

Comment:
isShallowEqual would work here too.

Reply:
I ended up not using it here  because both of these can be undefined which upsets onShallowEqual 🤷‍♂